### PR TITLE
fix(correction): When a translated item is corrected, send superdesk …

### PR DESCRIPTION
**Pull request by ioanpocol**
…and email notifications to translate creator and publisher (SDFID-514)

### CODE REVIEW

File: superdesk/templates/translation_changed_subject.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+The translated item {{title}} has been changed
```
Comment (petrjasek) [inline superdesk/templates/translation_changed_subject.txt]: imo we shouldn't use `title` here, either `headline` or just pass the item, to be consistent with other templates and UI

Comment (ioanpocol) [reply superdesk/templates/translation_changed_subject.txt]: I replaced title to headline. At list in this file there is no case to send the item to the template.

File: superdesk/templates/translation_changed.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% extends "email_layout.html" %}
+
+{% block content %}
+<p>{{username}} changed the following translated article: <a href="{{link}}">{{link}}</a></p>
```
Comment (petrjasek) [inline superdesk/templates/translation_changed.html]: imo here we could use item headline for the link text

Comment (ioanpocol) [reply superdesk/templates/translation_changed.html]: This was my first intention but then I seen that on user_mentions is used in this way.
I think that the scope was to be able to copy the link.

**Pull request merged**